### ISSUE TITLE
syntax error in ReasoningUtilities.py

### ISSUE THREAD

**Issue by edeutsch** [RTXteam/RTX#320]
Crash report:
  File "/mnt/data/orangeboard/devED/RTX/code/UI/OpenAPI/python-flask-server/openapi_server/../../../../reasoningtool/QuestionAnswering/ParseQuestion.py", line 3, in <module>
    import Question
  File "/mnt/data/orangeboard/devED/RTX/code/UI/OpenAPI/python-flask-server/openapi_server/../../../../reasoningtool/QuestionAnswering/Question.py", line 14, in <module>
    from ReasoningUtilities import ReasoningUtilities as RU
  File "/mnt/data/orangeboard/devED/RTX/code/UI/OpenAPI/python-flask-server/openapi_server/../../../../reasoningtool/QuestionAnswering/ReasoningUtilities.py", line 1905
    return f"'{value}'"
                      ^
SyntaxError: invalid syntax

I don't even understand what this syntax is supposed to do. Is this truly a syntax error? Is it possibly a syntax feature of a newer Python?

How should we resolve this. As is, RTX cannot launch on rtx.ncats.io/devED until this is fixed in branch NewFmt.

**Comment by edeutsch** [comment]
aha, after googling around, this appears to be use of f-strings. A new feature of Python 3.6. We are still running Python 3.5 on rtx.ncats.io

Issue #308 rises again.
Presumably the above code would work with modern Python.

I will push to upgrade Python.

**Comment by edeutsch** [comment]
The final status here is the original Python is still in place on rtx, but I have installed Python 3.7.3 in a different place (/mnt/data/python/Python-3.7.3) and all instance of RTX that run NewFmt and newer code will be configured to use that version of Python. Therefore both versions of Python will coexist (in peaceful terms, I hope). As a reminder, if you run NewFmt or newer, Python 3.6 or newer is required because of inclusion of code that uses f-strings. Closing this issue.
